code/style.css

```diff
@@ -1,5 +1,150 @@
-/* If you can see a peach background, the css file is correctly linked. Feel free to change or remove this styling! */
+@import
+url('https://fonts.googleapis.com/css2?family=Bungee+Hairline&family=Bungee+Shade&family=Quicksand&display=swap');
+
+/* 
+font-family: 'Bungee Hairline', cursive;
+font-family: 'Bungee Shade', cursive;
+font-family: 'Quicksand', sans-serif; 
+*/ 
 
 body {
-  background-color: peachpuff;
+  background-color: rgb(194, 216, 245);
+  font-family: 'Quicksand', sans-serif; 
+}
+
+header{
+  overflow: hidden;
+  width: 100%;
```

Comment:
I would suggest getting rid of either the padding or the width on the `header` and the `footer` because it's causing a horizontal scroll which we want to avoid when working with responsive web design.